probe-rs/src/architecture/riscv/mod.rs

```diff
@@ -28,12 +28,38 @@ pub mod sequences;
 /// A interface to operate RISC-V cores.
 pub struct Riscv32<'probe> {
     interface: &'probe mut RiscvCommunicationInterface,
+    state: &'probe mut RiscVState,
+}
+
+// Resume the core.
+fn resume_core(riscv32: &mut Riscv32) -> Result<(), crate::Error> {
```

Comment:
Could you make this a method of `Riscv32` please? :)

Reply:
Makes perfect sense :)